roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -79,7 +79,7 @@
     - { option: "name", value: "CentOS-{{ ansible_distribution_major_version }} - Extras" }
     - { option: "enabled", value: "1" }
     - { option: "gpgcheck", value: "0" }
-    - { option: "baseurl", value: "http://mirror.centos.org/centos/{{ ansible_distribution_major_version }}/extras/$basearch/" }
+    - { option: "baseurl", value: "http://mirror.centos.org/centos/{{ ansible_distribution_major_version }}/extras/$basearch/os/" }
```

Comment:
This seems valid as we can see repomd.xml under http://mirror.centos.org/centos/8/extras/x86_64/os/repodata/

/lgtm

Reply:
forgot to run the test..

/ok-to-test